MediaGalleryUi/Test/Mftf/ActionGroup/AdminOpenMediaGalleryForExistingPageNoEditorActionGroup.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminOpenMediaGalleryForExistingPageNoEditorActionGroup">
```

Comment:
Can you please replace existing usages of `AdminOpenMediaGalleryForPageNoEditorActionGroup` with opening a page and using this action group.

Also I'd correct a name as I believe this action group should work not only for the existing page
```suggestion
    <actionGroup name="AdminOpenMediaGalleryFromPageNoEditorActionGroup">
```